client/dev-tools/components/dev-tools.tsx

```diff
@@ -33,7 +33,9 @@ const PromoCard = ( { title, text, supportContext }: PromoCardProps ) => (
 );
 
 const DevTools = () => {
-	const [ showEligibility, setShowEligibility ] = useState( false );
+	const { searchParams } = new URL( document.location.toString() );
+	const showActivationModal = searchParams.get( 'activate' ) !== null;
+	const [ showEligibility, setShowEligibility ] = useState( showActivationModal );
```

Comment:
For some reason, a few lines below we check if the site can go Atomic before changing this to true:

https://github.com/Automattic/wp-calypso/blob/54c7136559f3e5fdc1f9dcedbb33c6988340b19c/client/dev-tools/components/dev-tools.tsx#L126-L128

Should we do the same here?

Reply:
> Should we do the same here?
Sorry, do what here?

That seems to double-check that the user can click the "Activate now" button to open the activation modal. cc: @arthur791004 maybe I'm missing something.

Reply:
I think it's fine. The `activate` is just to show the eligibility automatically. If the site cannot go atomic, we should not open the eligibility.